org-code-codebuilderapp/app/src/main/java/org/code/codebuilderapp/AnonymousHandshakeHandler.java

```diff
@@ -0,0 +1,30 @@
+package org.code.codebuilderapp;
+
+import java.security.Principal;
+import java.util.Map;
+import java.util.UUID;
+import org.springframework.http.server.ServerHttpRequest;
+import org.springframework.web.socket.WebSocketHandler;
+import org.springframework.web.socket.server.support.DefaultHandshakeHandler;
+
+/**
+ * Assignes a unique id to each handshake to allow messages to be directed
+ * to specific anonymous users.
+ */
+public class AnonymousHandshakeHandler extends DefaultHandshakeHandler {
```

Comment:
Placeholder until we have access control figured out.
Some discussion on this here: https://codedotorg.slack.com/archives/C03CK49G9/p1613502389142700?thread_ts=1613425248.140900&cid=C03CK49G9